exercises/practice/luhn/.meta/config.json

```
@@ -0,0 +1,21 @@
+{
+  "blurb": "Given a number determine whether or not it is valid per the Luhn formula.",
+  "authors": [
+    "LeaveNhA"
+  ],
+  "contributors": [
+  ],
+  "files": {
+    "solution": [
+      "src/luhn.cljs"
+    ],
+    "test": [
+      "test/luhn_test.cljs"
+    ],
+    "example": [
+      ".meta/src/example.clj"
```

I think you're missing the trailing s here.

```suggestion
      ".meta/src/example.cljs"
```